DataFormats/TCDS/src/classes_def.xml

```diff
@@ -8,8 +8,8 @@
     <version ClassVersion="4" checksum="3132515293"/>
   </class>
 
-  <class name="TCDSRecord" ClassVersion="3">
-    <version ClassVersion="3" checksum="1408516613"/>
+  <class name="TCDSRecord" ClassVersion="4">
+    <version ClassVersion="4" checksum="850266115"/>
```

Comment:
Please don't remove version 3, for back-compatibility

Reply:
I changed back ClassVersion to 3.

Reply:
> I changed back ClassVersion to 3.

Sorry @smorovic , I was not clear. You should list **both** versions, so that the new one is inserted, and the old one is kept for back compatibility:
```
  <class name="TCDSRecord" ClassVersion="4">
    <version ClassVersion="4" checksum="850266115"/>
    <version ClassVersion="3" checksum="1408516613"/>
  </class>
```

Reply:
Ok, no problem. Thanks for clarification. I included both.